web/src/ApiButton.tsx

```diff
@@ -281,7 +281,10 @@ export function ApiButton(props: React.PropsWithChildren<ApiButtonProps>) {
 
   const { setError } = useHudErrorContext()
 
-  const onClick = async () => {
+  const onClick = async (e: React.MouseEvent<HTMLElement>) => {
+    if ((e.target as Element).classList.contains("ignoreClick")) {
+      return true
+    }
```

Comment:
it seems a little fishy to me to cancel a button click if it happens on a certain area of a button. (to me, that's an indication that maybe html structure and css styling could be improved, and i'd be a little bit concerned with ux or a11y ux implications.) is there a reason you went with this approach vs a css layout adjustment?